.github/workflows/tests.yml

```diff
@@ -211,154 +215,129 @@ jobs:
           PACKAGE_NAME: ${{env.PACKAGE_NAME}}
           working-directory: src
 
-      - name: "Separate long Core tests"
-        shell: pwsh
+      - name: "Set tox extra CLI arguments"
+        id: tox-cli-arguments
+        shell: bash
         run: |
-          .github\workflows\scripts\separate_long_core_tests.ps1
+          if [ $MODE == 'PIP' ]; then
+            if [ ${{ matrix.os }} == 'ubuntu-latest' ]; then
+              echo 'TOX_EXTRA_ARG=--installpkg dist/${{ steps.wheel.outputs.wheel_name }} -x testenv.deps+="-e dpf-standalone/v${{inputs.ANSYS_VERSION}}" -x testenv.commands_pre+="uv pip install --extra-index-url https://wheels.vtk.org vtk-osmesa==9.2.20230527.dev0"' >> "$GITHUB_OUTPUT"
+            else
+              echo 'TOX_EXTRA_ARG=--installpkg dist/${{ steps.wheel.outputs.wheel_name }} -x testenv.deps+="-e dpf-standalone/v${{inputs.ANSYS_VERSION}}"' >> "$GITHUB_OUTPUT"
+            fi
+          else
+            if [ ${{ matrix.os }} == 'ubuntu-latest' ]; then
+              echo 'TOX_EXTRA_ARG=--installpkg dist/${{ steps.wheel.outputs.wheel_name }} -x testenv.commands_pre+="uv pip install --extra-index-url https://wheels.vtk.org vtk-osmesa==9.2.20230527.dev0"' >> "$GITHUB_OUTPUT"
+            else
+              echo 'TOX_EXTRA_ARG=--installpkg dist/${{ steps.wheel.outputs.wheel_name }}' >> "$GITHUB_OUTPUT"
+            fi
+          fi
 
-      - name: "Set pytest arguments"
+      - name: "Organize test files"
         shell: bash
         run: |
-          echo "COVERAGE=--cov=ansys.dpf.${{env.MODULE}} --cov-report=xml --cov-report=html --log-level=ERROR --cov-append" >> $GITHUB_ENV
-          echo "RERUNS=--reruns 2 --reruns-delay 1" >> $GITHUB_ENV
+          tox -e pretest
```

Comment:
@moe-ad so when run locally, do we need to manually run each of the following steps separately? Or can we use a single command?